.github/workflows/bump_dependencies.yml

```diff
@@ -0,0 +1,53 @@
+name: Bump Versions
+
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  schedule:
+    - cron: '30 17 * * WED' # 12:30 EST on wednesdays
+  workflow_dispatch:
+
+jobs:
+  bump-versions:
+    name: Bump Package Versions
+    if: github.repository_owner == 'viamrobotics'
+    runs-on: [self-hosted, x64]
+    container:
+      image: ghcr.io/viamrobotics/canon:amd64
+    timeout-minutes: 10
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.head_ref }}
+        token: ${{ secrets.GIT_ACCESS_TOKEN }}
```

Comment:
```suggestion
```


I don't think this is required. You only need a ref when checking out additional code or branches that aren't what triggered the workflow in the first place. Likewise, the token shouldn't be needed since the default token can always read the repo for checkout.